packages/slate-theme/src/sections/header.liquid

```diff
@@ -78,31 +78,41 @@
   </header>
 
   <nav role="navigation">
-    <ul class="site-nav">
+    <ul>
       {% for link in linklists[section.settings.main_linklist].links %}
-        {%- assign child_list_handle = link.title | handleize -%}
-        {% if menus[child_list_handle].links != blank %}
-          <li class="site-nav--has-submenu{% if link.active %} site-nav--active{% endif %}">
-            <a href="{{ link.url }}" class="site-nav__link">
-              {{ link.title }}
-              {% include 'icon-arrow-down' %}
-            </a>
-            <ul class="site-nav__submenu">
-              {% for childlink in menus[child_list_handle].links %}
-                <li {% if childlink.active %}class="site-nav--active"{% endif %}>
-                  <a href="{{ childlink.url }}" class="site-nav__link">{{ childlink.title | escape }}</a>
-                </li>
+        {% if link.links != blank %}
+          <li>
```

Comment:
Instead of having the `if` condition right after the `for` loop, we can extract the `<li>` so we don't need to rewrite it in the `{% else %}` block:

```liquid
{% for link in linklists[section.settings.main_linklist].links %}
  <li>
    {% if link.links != blank %}
      ...
    {% else %}
      <a href="{{ childlink.url }}">{{ childlink.title | escape }}</a>
    {% endif %}
  </li>
{% endfor %}
```

Reply:
I'm not sure what @t-kelly is suggesting is suppose to replace 🤔

Reply:
After discussion, we'll keep it inside.